src/main/java/com/redhat/management/approval/EmailBody.java

```diff
@@ -170,6 +176,26 @@ private static String getUrlContent(URL url) throws Exception {
         return content.toString();
     }
 
+    private String encrypt(String strToEncrypt) throws Exception {
+        String v2Key = System.getenv("ENCRYPTION_KEY");
+        if(v2Key == null) {
+            v2Key = "5ysYUd3Qrjj7DDplmEJHmnrFBEPS887JwOQv0jFYq2g=";
+        }
+        byte[] keyBytes = Base64.getDecoder().decode(v2Key);
+        SecretKeySpec secretKey = new SecretKeySpec(keyBytes, "AES");
+
+        byte[] iv = { 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0 };
+        IvParameterSpec ivpec = new IvParameterSpec(iv);
+
+        Cipher cipher = Cipher.getInstance("AES/CBC/PKCS5Padding");
+        cipher.init(Cipher.ENCRYPT_MODE, secretKey, ivpec);
+
+        String output = Base64.getEncoder().encodeToString(cipher.doFinal(strToEncrypt.getBytes("UTF-8")));
```

Comment:
@bzwei Should you be using the URL Encoder which doesn't add new lines
https://docs.oracle.com/javase/8/docs/api/java/util/Base64.html

Reply:
I followed the way how `ManageIQ::Password` encrypts: https://github.com/ManageIQ/manageiq-password/blob/master/lib/manageiq/password.rb#L26. It has to be decrypt precisely in our ruby code.